analysis/dataset_definition_controls.py

```diff
@@ -0,0 +1,16 @@
+from ehrql import create_dataset
+from ehrql.tables.core import clinical_events, patients
+
+
+start_date = "2020-01-01"
+
+dataset = create_dataset()
+dataset.configure_dummy_data(population_size=1000)
+
+dataset.sex = patients.sex
+dataset.age = patients.age_on(start_date)
+dataset.indexdate = (
+    clinical_events.sort_by(clinical_events.date).first_for_patient().date
+)
+
```

Comment:
should the controls dataset definition specifically exclude people with the diagnoses mentioned in the cases dataset definition?

Reply:
(maybe this is only for CI, but people might use it as a template if they come across it)

Reply:
cool thanks